### DISCUSSION TITLE
Is Android still supported?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [ActivityWatch/activitywatch#953] [Q&A] [answered]
*   https://github.com/orgs/ActivityWatch/discussions/952#discussion-5660157
*   https://forum.activitywatch.net/t/f-droid-support/690/2?u=rokejulianlockhart
*   https://github.com/ActivityWatch/activitywatch/releases/tag/v0.12.3b10 hasn't even published an `.apk` file.

**Comment by ErikBjare** [comment, marked as answer]
We've had trouble shipping a new release due to persistent build issues: https://github.com/ActivityWatch/aw-android/issues/93
